conf/modules.config

```diff
@@ -1,84 +1,377 @@
+            "-enzyme '${params.enzyme}'",
+            "-spectrum_batch_size ${params.spectrum_batch_size}"
+        ].join(' ').trim()
+        publishDir  = [ enabled: false ]
+    }
+
+    withName: OPENMS_IDFILTER_FOR_ALIGNMENT {
+        ext.args    = [
+            "-remove_decoys",
+            "-precursor:length '${params.peptide_min_length}:${params.peptide_max_length}'",
+            "-delete_unreferenced_peptide_hits",
+            (params.fdr_threshold == '0.01') ? "-score:pep 0.05" : "-score:pep " + params.fdr_threshold
+        ].join(' ').trim()
+        publishDir  = [
+            path: { "${params.outdir}/Intermediate_Results" },
+            mode: 'copy',
+            pattern: '*.idXML'
+        ]
+    }
+
+    withName: OPENMS_IDFILTER_Q_VALUE {
+        ext.prefix  = { "${meta.id}_filtered" }
+        ext.args    = [
+            "-remove_decoys",
+            "-precursor:length '${params.peptide_min_length}:${params.peptide_max_length}'",
+            "-delete_unreferenced_peptide_hits",
+            (params.fdr_threshold == '0.01') ? "-score:pep 0.05" : "-score:pep " + params.fdr_threshold
+        ].join(' ').trim()
+        publishDir  = [
+            path: { "${params.outdir}/Intermediate_Results" },
+            mode: 'copy',
+            pattern: '*.idXML'
+        ]
+    }
+
+    withName: OPENMS_PERCOLATORADAPTER {
+        ext.prefix  = { "${meta.id}_all_ids_merged_psm_perc" }
+        ext.args    = [
+            "-seed 4711",
+            "-trainFDR 0.05",
+            "-testFDR 0.05",
+            "-enzyme no_enzyme",
+            "-subset_max_train ${params.subset_max_train}",
+            "-doc ${params.description_correct_features} ",
+            (params.fdr_level != 'psm-level-fdrs') ? "-" + params.fdr_level : ""
+        ].join(' ').trim()
+        publishDir  = [
+            path: { "${params.outdir}/Intermediate_Results" },
+            mode: 'copy',
+            pattern: '*.idXML'
+        ]
+    }
+
+    withName: OPENMS_PSMFEATUREEXTRACTOR {
+        publishDir  = [
+            path: { "${params.outdir}/Intermediate_Results" },
+            mode: 'copy',
+            pattern: '*.idXML'
+        ]
+    }
+
+    withName: 'OPENMS_DECOYDATABASE|OPENMS_THERMORAWFILEPARSER|OPENMS_PEAKPICKERHIRES|OPENMS_PEPTIDEINDEXER|OPENMS_FALSEDISCOVERYRATE|OPENMS_IDMERGER|OPENMS_MAPRTTRANSFORMER.*ML|OPENMS_FEATURE.*|OPENMS_IDCONFLICTRESOLVER|OPENMS_MZTABEXPORTER' {
+        publishDir       = [ enabled: false ]
+    }
+
+    withName: 'OPENMS_TEXTEXPORTER' {
+        ext.args    = [
+            "-id:add_metavalues 0"
+        ].join(' ').trim()
+        publishDir  = [
+            path: { "${params.outdir}/" },
+            mode: 'copy',
+            pattern: '*.tsv'
+        ]
+    }
+
+    withName: 'OPENMS_TEXTEXPORTER_PSMS' {
+        publishDir  = [
+            path: { "${params.outdir}/PSMs" },
+            mode: 'copy',
+            pattern: '*.tsv'
+        ]
+    }
```

Comment:
If I see this right, those two processes are included from the same module:
in subworkflows/local/post_quantification.nf
`include { OPENMS_TEXTEXPORTER }                                             from '../../modules/local/openms_textexporter'`
in subworkflows/local/pre_quantification.nf
`include { OPENMS_TEXTEXPORTER as OPENMS_TEXTEXPORTER_PSMS }                 from '../../modules/local/openms_textexporter'`
I am not sure if you are aware that 
```
        ext.args    = [
            "-id:add_metavalues 0"
        ].join(' ').trim()
```
are used by `OPENMS_TEXTEXPORTER` and `OPENMS_TEXTEXPORTER_PSMS` the way you did it. Just in case this is not intended.

Reply:
So how would you recommmend to do this when I only want the process to run with the OPENMS_TEXTEXPORTER, just rename the original name?

Reply:
I see 2 solutions:
- overwrite ext.args in OPENMS_TEXTEXPORTER_PSMS, con: whatever you add new to OPENMS_TEXTEXPORTER in future might get unintended also to OPENMS_TEXTEXPORTER_PSMS
- always use `include { OPENMS_TEXTEXPORTER as <name> } from '../../modules/local/openms_textexporter'` and never have OPENMS_TEXTEXPORTER in the modules.config